pkg/gatewayserver/io/io.go

```diff
@@ -602,6 +602,7 @@ func (c *Connection) ScheduleDown(path *ttnpb.DownlinkPath, msg *ttnpb.DownlinkM
 		} else {
 			settings.Timestamp = 0
 		}
+		settings.ConcentratorTimestamp = uint64(time.Duration(em.Starts()))
```

Comment:
We don't need to convert this to microseconds since the LBS frontend will cast it to `scheduling.ConcentratorTime` and that's nanoseconds.

Reply:
You can use `int64(em.Starts())` directly then, right ?